ports/mimxrt/mpconfigport.h

```diff
@@ -29,7 +29,7 @@
 // Board specific definitions
 #include "mpconfigboard.h"
 #include "fsl_common.h"
-#include "lib/nxp_driver/sdk/CMSIS/Include/core_cm7.h"
+#include CMSIS_HEADER_H
```

Comment:
I tried removing this line altogether, and the boards seem to build fine.  I also checked TEENSY40 before and after removing this line, and the firmware is equivalent.

So, I think it's best to just remove it.

@robert-hh is it safe to remove this header include?